src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,152 @@
 package core.basesyntax;
 
+import java.util.Map;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int DEFAULT_SIZE_MULTIPLAYER = 2;
+    private int loadFactor = 12;
```

Comment:
* this looks more like a threshold if size will pass it - then resize
* load factor's default value = 0.75f, and it is how you get this 12 number, so let's add and use it properly too
```suggestion
    private int threshold = 12;
```